### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by Newtayo**
**In this pull request, I implemented the following features**

- [ ]  Designed the World Population App Set up
- [ ] Fetched data from the restcountries API to be rendered on the Application
- [ ] Created a redux store for storing the data stored in the API
- [ ] Added functionality for displaying the list of countries in the world as well as displaying peculiar information.
- [ ] Web Application was deployed on render

### CODE REVIEW

File: src/__test__/Countrylist.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const tree = renderer
+    .create(
+      <Provider store={Store}>
+        <BrowserRouter>
+          <Countrylist />
+        </BrowserRouter>
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```
Comment (Munsa1) [inline src/__test__/Countrylist.test.js]: - [ ] Its a requirement that the app has unit tests for several components of the app and some pure functions in Redux code.
- [ ] Also the App should have integration tests using React Testing Library, you can perform a mocking data fetching and Redux Store connection.

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
```
Comment (Munsa1) [inline src/index.css]: - [ ] Great job following the Layout: composition and space between elements in the given [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

- [ ] Kindly use the given typography: font face
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Newtayo,  Nice video presentation, you were audible and profesional and highlighted what you found interesting about your project

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Newtayo** [comment]
**In this Pull Request, I Implemented the following**

- [ ] Implemented test for all the components
- [ ] Adopted Lato font for the web app

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Newtayo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**